src/components/EntityInfo/EntityMenu.vue

```diff
@@ -1,16 +1,40 @@
 <template>
   <menu id="entity-menu">
     <!-- Staff Comments -->
-    <span v-if="isAllowed(AllowableActions.STAFF_COMMENT)">
+    <span v-if="!isDisableNonBenCorps && isAllowed(AllowableActions.STAFF_COMMENT)">
       <StaffComments
         :axios="axios"
         :businessId="businessId"
         :maxLength="2000"
       />
     </span>
 
+    <!-- COLIN link button -->
+    <span v-if="isDisableNonBenCorps && isBusiness">
+      <v-tooltip
+        top
+        content-class="top-tooltip"
+        transition="fade-transition"
+      >
+        <template #activator="{ on }">
+          <v-btn
+            id="colin-link-button"
+            small
+            text
+            color="primary"
+            :href="getCorporateOnlineUrl"
```

Comment:
By implementing thusly (instead of calling a method that redirects to the COLIN URL), this becomes a "link button" and you can open it in a new tab/window:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/b2d33d0d-d0cf-4109-9d54-95035b1ce082)